src/main/java/edu/kit/ipd/crowdcontrol/objectservice/rest/RequestUtil.java

```diff
@@ -5,6 +5,10 @@
 import spark.Request;
 
 public class RequestUtil {
+    private RequestUtil() {
+        // utility class
+    }
```

Comment:
utility class are still not always static. And they are not generated when you export to html, but the IDE still shows them. But i was talking more about the class-javadoc. something like:
`\** bla bla contains static methods to help with Sparkjava-requests....`